### PR TITLE
Order invoice price incl tax check

### LINKED ISSUE #294: In order invoice PDF unit price, item price , subtotal amount and discount price is configured with including tax property's, This leads confusion to customer.

### PR DESCRIPTION
**Pull request by jithendraprasad9999**
Resolves #294  #295 
Type: **bugfix**

## Issue
1.In order invoice PDF unit price, item price , subtotal amount and discount price is configured with including tax property's, This leads confusion to customer. 
2. Header text and footer are coming with some junk characters in PDF.

## Solution
1.For all the above prices added check to include or exclude tax based on CustomerTaxDisplayTypeId from order details . 
2.Decoded header text and footer text with Html.Raw attribute to avoid junk characters. 

## Breaking changes
None

## Testing
1. Place an order , in order history page select respective order and tap on details button . And then  click on "Print invoice button"
2.  Based on TaxDisplayType setting, prices will get populate either with including tax or excluding tax

### CODE REVIEW

File: src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 @inject Grand.Business.Core.Interfaces.Common.Directory.ICurrencyService _currencyService
 @inject Grand.Business.Core.Interfaces.Checkout.GiftVouchers.IGiftVoucherService _giftVoucherService
-
+@inject IWorkContext workContext
 @{
     var logoPicture = await _pictureService.GetPictureById(_pdfSettings.LogoPictureId);
     var logoExists = logoPicture != null;
-    var limitbyVendor = !string.IsNullOrEmpty(Model.Item2);
+    var limitbyVendor = !string.IsNullOrEmpty(Model.Item2); 
+    var priceIncludesTax = workContext.TaxDisplayType == TaxDisplayType.IncludingTax;
```
Comment (KrzysztofPajak) [inline src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml:16]: TaxDisplayType should get from the Order

File: src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml [hunk 2]
```diff
@@ -87,6 +88,7 @@
                                 string paymentMethodStr = paymentMethod != null ? paymentMethod.FriendlyName : order.PaymentMethodSystemName;
                             }
                             <li>@(string.Format(_translationService.GetResource("PDFInvoice.PaymentMethod", order.CustomerLanguageId), paymentMethodStr))</li>
+                            @await Component.InvokeAsync("Widget", new { widgetZone = "payment_method_below", additionalData = @order.Id })
```
Comment (KrzysztofPajak) [inline src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml:91]: Please change the name for the widget zone, better name will be i.e: pdf_payment_method_below

File: src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml (hunk 1 above)
Comment (jithendraprasad9999) [reply src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml]: @KrzysztofPajak updated code as per feedback

File: src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml (hunk 2 above)
Comment (jithendraprasad9999) [reply src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml]: @KrzysztofPajak updated the zone name

**Comment by KrzysztofPajak** [comment]
Thanks for contributing!

**Pull request merged**